plugins/anchor-links/index.js

```diff
@@ -6,6 +6,10 @@ const stringify = require('remark-stringify')
 
 // This plugin adds anchor links to headlines and lists that begin with inline
 // code blocks.
+//
+// NOTE: Some of the HTML code is duplicated in:
+//       https://github.com/hashicorp/consul/blob/517cfa79184dc2e43bd61007d4d76946f08591d3/website/components/config-entry-reference/index.jsx#L118-L136
```

Comment:
Will update with right link once that PR is merged.

Reply:
I'll hold off until after merge to run this one then? Or do you have any opinion on the ordering?

Reply:
Yeah hold off for the merge until I can get the permalink from the master branch.

Reply:
@jescalan this is ready to merge. I've updated the permalink.